machine/v1/types_powervsprovider.go

```diff
@@ -11,6 +11,15 @@ type PowerVSResourceType string
 // PowerVSProcessorType enum attribute to identify the PowerVS instance processor type
 type PowerVSProcessorType string
 
+// IBMVPCLoadBalancerType is the type of LoadBalancer to use when registering
+// an instance with load balancers specified in LoadBalancerNames
+type IBMVPCLoadBalancerType string
+
+// ApplicationLoadBalancerType is possible values for IBMVPCLoadBalancerType.
+const (
+	ApplicationLoadBalancerType IBMVPCLoadBalancerType = "application" // Application Load Balancer for VPC (ALB)
```

Comment:
Kube conventions dictate that enums should use PascalCase, I would expect this to be `Application` and to be converted to the IBM version within the controller

Reply:
Updated to use it as Application.